http/action/src/test/java/io/knotx/databridge/http/action/HttpActionTest.java

```diff
@@ -442,7 +447,7 @@ void testSuccessTransitionExpectedAndResponseAsRawText(String contentType, boole
 
   @ParameterizedTest(name = "Expect error transition and no response")
   @MethodSource("dataExpectErrorTransitionAndNullBody")
-  void testErrorResponseAndNoResponse(String contentType, boolean forceJson, String jsonPredicate,
+  void testErrorTransitionAndNoResponse(String contentType, boolean forceJson, String jsonPredicate,
```

Comment:
Can we please add one case scenario:
- some exception is thrown
- incoming fragment is not changed - it means that it still contains its payload and body - you can validate the whole fragment object.